intermediate_source/pruning_tutorial.py

```diff
@@ -365,38 +324,31 @@ def compute_mask(self, t, default_mask):
         return mask
 
 ######################################################################
-# Now, to apply this to a parameter in an ``nn.Module``, you should
-# also provide a simple function that instantiates the method and
-# applies it.
+# ``nn.Module``의 매개변수에 적용하기 위해 인스턴스화하고 적용하는 간단한 기능을 구현해봅니다.
 def foobar_unstructured(module, name):
-    """Prunes tensor corresponding to parameter called `name` in `module`
-    by removing every other entry in the tensors.
-    Modifies module in place (and also return the modified module) 
-    by:
-    1) adding a named buffer called `name+'_mask'` corresponding to the 
-    binary mask applied to the parameter `name` by the pruning method.
-    The parameter `name` is replaced by its pruned version, while the 
-    original (unpruned) parameter is stored in a new parameter named 
-    `name+'_orig'`.
-
-    Args:
-        module (nn.Module): module containing the tensor to prune
-        name (string): parameter name within `module` on which pruning
-                will act.
-
-    Returns:
-        module (nn.Module): modified (i.e. pruned) version of the input
-            module
+    """
+    텐서 내 다른 모든 항목들을 제거하여 `module`에서 `name` 이라는 파라미터에 대해 가자치기 기법을 적용
+    다음 내용에 따라 모듈을 수정 (또는 수정된 모듈을 반환):
+        1) 가지치기 기법에 의해 매개변수 `name`에 적용된 이진 마스크에 해당하는 명명된 버퍼 `name+_mask` 를 추가합니다.
+        `name` 파라미터는 가지치기 기법이 적용된 것으로 대체되며, 가지치기 기법이 적용되지 않은 
+        기존의 파라미터는 `name+'_orig'` 이름이라는 새로운 매개변수에 저장됩니다.
```

Comment:
요렇게 고치면 조금 더 읽기 수월하지 않을까요?

```suggestion
        기존의 파라미터는 `name+'_orig'` 라는 이름의 새로운 매개변수에 저장됩니다.
```

Reply:
네! 좋은 의견인 것 같습니다ㅎㅎ 수정하겠습니다!